browser/ui/views/brave_help_bubble/brave_help_bubble_delegate.cc

```diff
@@ -0,0 +1,234 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/ui/views/brave_help_bubble/brave_help_bubble_delegate.h"
+
+#include <utility>
+
+#include "components/grit/brave_components_strings.h"
+#include "ui/base/l10n/l10n_util.h"
+#include "ui/base/metadata/metadata_impl_macros.h"
+#include "ui/gfx/canvas.h"
+#include "ui/gfx/scoped_canvas.h"
+#include "ui/views/border.h"
+#include "ui/views/bubble/bubble_border.h"
+#include "ui/views/bubble/bubble_border_arrow_utils.h"
+#include "ui/views/bubble/bubble_frame_view.h"
+#include "ui/views/controls/label.h"
+#include "ui/views/layout/fill_layout.h"
+#include "ui/views/widget/widget_delegate.h"
+
+using views::BubbleBorder;
+using views::BubbleDialogDelegateView;
+using views::BubbleFrameView;
+using views::NonClientFrameView;
+
+namespace {
+constexpr SkColor kBgColor = SkColorSetARGB(0xFF, 0x20, 0x4A, 0xE3);
+
+// This class paints a border with only an arrow.
```

Comment:
It is my understanding that we want to skip paiting shadow and border, isn't it? could you write this to the comment?